android/src/main/java/com/thebluealliance/androidclient/adapters/MatchListAdapter.java

```diff
@@ -34,6 +34,11 @@ public void setTeamKey(String teamKey) {
 
     @Override
     public View getChildView(int groupPosition, int childPosition, boolean isLastChild, View convertView, ViewGroup parent) {
+        if (groupPosition >= mGroups.size()
```

Comment:
I'll file a new issue so I don't forget to check this out in the future. This is a good enough fix for now; I'd rather have sketchy rendering behavior than crashes!